assets/scripts/script.js

```diff
@@ -1,27 +1,32 @@
-// const check = document.getElementById("check");
-// const displayy = document.getElementById("display");
+const check = document.getElementById('check')
+const displayy = document.getElementById('display')
 
-// function palindrom() {
-//   let enterr = document.getElementById("start").value;
+function palindrom () {
+  const enterr = document.getElementById('start').value
+  let result = ''
+  if (enterr === '') {
+    displayy.innerHTML = ' type a text or number'
+  } else {
+    const restrictSymbols = enterr.replace(/[^a-z0-9]/g, '').toLowerCase()
+    result = restrictSymbols.split('').reverse().join('')
 
-//   let result = "";
-//   result = enterr.split("").reverse().join("");
+    // console.log('this is result', result)
+    // console.log('this is input', enterr)
 
-//   // console.log('this is result', result)
-//   // console.log('this is input', enterr)
+    if (restrictSymbols === result) {
+      displayy.innerHTML = 'it is a palindrome'
+      console.log('it is a palindrome')
 
-//   if (enterr === result) {
-//     displayy.innerHTML = "it is a palindrome";
-//     console.log("it is a palindrome");
+      // return "it is a palindrome"
+    } else {
+      displayy.innerHTML = 'it is not a palindrome'
+      console.log('it is not a palindrome')
```

Comment:
I'm guessing your consoles are for debugging purposes? remove them in the final submission